packages/pwa-kit-react-sdk/src/ssr/browser/main.jsx

```diff
@@ -118,8 +119,14 @@ export const start = () => {
     return Promise.resolve()
         .then(() => new Promise((resolve) => loadableReady(resolve)))
         .then(() => {
-            ReactDOM.hydrate(<OuterApp {...props} />, rootEl, () => {
-                window.__HYDRATING__ = false
-            })
+            hydrateRoot(
```

Comment:
API has changed for React 18: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/925/files#diff-f4ee7217dbf638680d6780866b7e72f56790717a5c6cc3ed1c982794a180ae38